docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -67,17 +91,43 @@ You must reload the page after disabling extensions or display issues may occur.
 
 :::
 
-## Rolling Back Extensions
+## Deleting Helm Charts
 
-Under the **Installed** tab, click the **Rollback** button on the extension you wish to roll back.
+1. Click **☰**, then click on the name of your local cluster.
+1. From the sidebar, select **Apps > Installed Apps**.
+1. Find the name of the chart you want to delete and select the checkbox next to it. 
+1. Click **Delete**.
```

Comment:
Why is this section here? You shouldn't need to delete an extension chart from the Installed Apps page.